From 62fc436a4f2b1ed3adbab920cf3b1572a5c53e9b Mon Sep 17 00:00:00 2001
From: anthraxx <levente@leventepolyak.net>
Date: Tue, 6 Dec 2016 20:39:04 +0100
Subject: [PATCH] extend global CFLAGS and LDFLAGS to aid distro packaging

This preserves globally defined CFLAGS and LDFLAGS and simply
extends those variables to aid distro based packaging toolchains
and predefined distro wide defaults like SSP, relro etc.

this re-fixes 7f8aaf74302816d03fbff62dab5c987d498acdde after it was
somehow undone ;)
---
 src/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/Makefile b/src/Makefile
index 308ad41..e7d8307 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -101,7 +101,7 @@ BINARY_NATIVE           := $(PROG_NAME)
 ## General compiler and linker options
 ##
 
-CFLAGS                  := -pipe -std=c99 -Iinclude/ -IOpenCL/
+CFLAGS                  += -pipe -std=c99 -Iinclude/ -IOpenCL/
 CFLAGS                  += -W
 CFLAGS                  += -Wall
 CFLAGS                  += -Wextra
@@ -137,7 +137,7 @@ CFLAGS                  += -ftrapv
 
 #CFLAGS                  += -Wstack-usage=524288
 
-LFLAGS                  :=
+LFLAGS                  := $(LDFLAGS)
 
 ifndef DEBUG
 CFLAGS                  += -O2
-- 
2.10.2

